### PR TITLE
Slebew

### PR DESCRIPTION
**Pull request by ArinoJenynof**
Ternyata gk sampe jam 1 hehe

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -45,81 +53,34 @@ client.on('ready', () => {
 client.on("interactionCreate", async (interaction) => {
   if (!interaction.isChatInputCommand()) return;
 
-  if (interaction.commandName === "lucu") {
-    await interaction.reply("sePong!");
-  }
+  const command = interaction.client.commands.get(interaction.commandName);
```
Comment (fadhlanmr) [inline main.js:56]: line sebanyak itu jadi 1 
image

**Comment by fadhlanmr** [comment]
gacor mang LGTM

**Review by mhdfadlyhasan — APPROVED** [review]

**Review by fadhlanmr — APPROVED** [review]

File: main.js (hunk 1 above)
Comment (mhdfadlyhasan) [reply main.js:56]: namanya juga di pindah, gimana sih lu

Comment (ArinoJenynof) [reply main.js:56]: Sausnya bang

https://discordjs.guide/creating-your-bot/command-handling.html

Comment (mhdfadlyhasan) [reply main.js:56]: too long wont read

**Pull request merged**